packages/cli/src/backend-identifier/local_namespace_resolver.ts

```diff
@@ -18,6 +18,8 @@ export class LocalNamespaceResolver implements NamespaceResolver {
    * Returns the value of package.json#name from the current working directory
    */
   resolve = async () => {
-    return (await this.packageJsonLoader.loadCwdPackageJson()).name;
+    const name = this.packageJsonLoader.read().name;
```

Comment:
if we call it `Loader` then `load` is fine. We can also name it reader.

Reply:
updated